batchflow/models/torch/visualization.py

```diff
@@ -287,6 +287,58 @@ def get_gradcam(self, *args, targets=None, feed_dict=None,
         camera = np.uint8(camera * 255)
         return camera
 
+    # Visualize signal propagation: activations statistics
+    def get_signal_propagation(self, model, image_shape, batch_size=3):
+        """ Compute signal propagation statistics of all layers in the network.
+
+        Under the hood, forward hooks are registered to capture outputs of all layers,
+        and they are removed after extraction of all the activations.
+        
+        ## Link and params"""
+        statistics = {
+            'Average Channel Squared Mean': [], 
+            'Average Channel Variance': []
+        }
+        modules_instances = []
```

Comment:
What is the reason for this list here?
Anyways, you can just make it as an additional key in your dictionary